src/services/state-persistor/state-persistor-rules-write.helper.ts

```diff
@@ -20,6 +21,16 @@ export abstract class RulesWriteHelper extends StatePersistorRulesHelper {
     if (key === SP_KEY_LAYERS_OPEN) {
       return SP_AVAILABLE_STORAGES.localStorage
     }
+    if (
+      key === 'basemap_2015_global' ||
+      key === 'topogr_global' ||
+      key === 'topo_bw_jpeg'
+    ) {
+      return SP_AVAILABLE_STORAGES.localStorage
+    }
+    if (key === SP_KEY_SERIAL) {
+      return SP_AVAILABLE_STORAGES.permalink
```

Comment:
I might be missing something, but I don't see any `serial` param added to the URL when modifying the style.

Reply:
fixed